cmd/mobile-security-service/main.go

```diff
@@ -23,12 +24,16 @@ func init() {
 
 func main() {
 	config := config.Get()
-	e := echo.New()
+	staticFilesDir := config.StaticFilesDir
+	apiRoutePrefix := config.ApiRoutePrefix
+
+	//e := echo.New()
+	e := web.NewRouter(staticFilesDir, apiRoutePrefix)
 
 	// Load middleware
 	middleware.Init(e, config)
```

Comment:
Move this, based on comment in router.go

Reply:
working on this now